others/kelvin_to_celsius.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief Conversion from [Kelvin to
+ * Celsius](https://byjus.com/chemistry/kelvin-to-celsius/) degrees.
+ * @details
+ * The algorithm consists on converting a Kelvin degree value to a Celsius
+ * value.
+ * The formula to convert a Kelvin to a Celsius value is:
+ * @f[ C = K - 273.15 @f] where:
+ * - C is the Celsius temperature
+ * - K is the Kelvin temperature
+ *
+ * Check out [Kelvin](https://en.wikipedia.org/wiki/Kelvin) and
+ * [Celsius](https://en.wikipedia.org/wiki/Celsius) on Wikipedia for more
+ * information about their story, how do they work, when and why they should be
+ * used, etc..
+ * @author [David Leal](https://github.com/Panquesito7)
+ */
+
+#include <cassert>   /// for assert
+#include <cmath>     /// for std::abs
+#include <iostream>  /// for IO operations
```

Comment:
```suggestion
#include <iostream>  /// for std::cout
```

Reply:
@realstealthninja please have a look [here](https://github.com/TheAlgorithms/C-Plus-Plus/pull/2429#discussion_r1092422377).

Reply:
I/O operations cout is an output operation it makes sense to me. You have to understand that if you want a change you don't do it through precedents because precedents are repelled and lost to time. If you want change create a discussion on it and if enough people agree it will be written into contribution.md. i am merely a man of the document it would be a much more useful debate if it was on the discussion board

Reply:
Personally, I don't think either should exist 🤷🏼. It is perfectly clear to every c++ programmer that starts out with a "Hello world" program what `iostream` is 😄 (just my 2 pennies)